src/api/app/helpers/webui/notification_helper.rb

```diff
@@ -40,4 +33,8 @@ def notification_filter_active?(filter_item)
       filter_item[:type] == 'unread'
     end
   end
+
+  def determine_badge_color(filter_item)
```

Comment:
I would use just `badge_color`.

Reply:
done

Reply:
yes, now you said it, the naming felt bad :D